### PR TITLE
[RND-193] Deployment .http test script, and some AWS touchup

### PR DESCRIPTION
**Pull request by bradbanister**

### CODE REVIEW

File: Meadowlark-js/yarn.lock [hunk 1]
```diff
@@ -3638,13 +3638,6 @@
   resolved "https://www.myget.org/F/ed-fi/npm/@tootallnate/once/-/@tootallnate/once-1.1.2.tgz#ccb91445360179a04e7fe6aff78c00ffc1eeaf82"
   integrity sha1-zLkURTYBeaBOf+av94wA/8Hur4I=
 
-"@types/async-retry@^1.4.2":
```
Comment (bradbanister) [inline Meadowlark-js/yarn.lock:3641]: Maybe meadowlark-faker removal didn't update this?

Comment (stephenfuqua) [reply Meadowlark-js/yarn.lock:3641]: I didn't think to re-lock at this level after deleting meadowlark-faker.

**Review by stephenfuqua — APPROVED** [review]

File: Meadowlark-js/packages/meadowlark/test/http/remote.deployment.test.http [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+# Ensure you have followed the DEPLOYMENT.md instructions. You should have deployed with ACCESS_TOKEN_REQUIRED=true
+# and have loaded descriptors with "yarn load:descriptors:aws-dev".
+
+# **** Change this to be the hostname portion of the endpoint URLs displayed at the end of the deployment process.
+@hostname=abcdefg.execute-api.us-east-1.amazonaws.com
+
+
+# FYI, this is a non-expiring bearer token that is valid in the default configuration for the OAuth 2.0 proof-of-concept. 
+@token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInN1YiI6InN1cGVyLWdyZWF0LVNJUyIsImp0aSI6IjNkNTliNzVmLWE3NjItNGJhYS05MTE2LTE5YzgyZmRmOGRlMyIsImlhdCI6MTYzNjU2MjA2MCwiZXhwIjozODQ1NTQ4ODgxfQ.WF5ABFZAvNsLDZktwa8VxDR846fGptRXJObtQitbL8I
```
Comment (stephenfuqua) [inline Meadowlark-js/packages/meadowlark/test/http/remote.deployment.test.http:21]: I'm going to make three changes here:

1. Adding `@stage1` token
2. Adding a call to the base API endpoint to "verify service is running"
3. Changing from `http` to `https`.

File: Meadowlark-js/packages/meadowlark/test/http/remote.deployment.test.http [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+
+### Attempt to delete the school - should succeed
+### (Cold start for DELETE means this may take up to 5 seconds.)
+DELETE http://{{hostname}}/dev/v3.3b/ed-fi/schools/7a5cf3f4a68015c0922e24c73401a21e9fd1767ef60c0b3300f2301e
+Authorization: bearer {{token}}
+
+
+### View all schools - should be none
+GET http://{{hostname}}/dev/v3.3b/ed-fi/schools
```
Comment (stephenfuqua) [inline Meadowlark-js/packages/meadowlark/test/http/remote.deployment.test.http:158]: I just saw eventual consistency in action:

1. DELETED - saw 204
2. Very quickly hit this next GET - returned a record
3. Then hit the GET again - empty array.

Fascinating! I'm adding a comment about this.

**Pull request merged**